.github/workflows/pr_check_workflow.yml

```diff
@@ -176,7 +176,7 @@ jobs:
         working-directory: ./artifacts
     strategy: 
       matrix:
-        version: [ osd-1.3.2,osd-2.0.0,osd-2.1.0 ]
+        version: [ osd-2.0.0, osd-2.1.0, osd-2.2.0 ]
```

Comment:
Why are we dropping the testing for 1.3.2? If this is a backport, shouldn't we just be adding 2.2.0 to the end of the list?

Reply:
+1

Reply:
great question. Compare https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2281/files and https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2148/files . We could see osd-1.3.2 is removed earlier. I think the problem is that we have build_and_test_workflow.yml in the lastest main but 2.x still use pr_check_workflow.yml (this still have v1.3.2). it seems we are missing some PRs in 2.x